doc/user/cluster_tls.md

```diff
@@ -1,15 +1,15 @@
-# Cluster TLS guide
+# Cluster TLS policy
```

Comment:
Policy

Reply:
it should be policy, not Policy. We do not capitalize word in headlines.